File: mtt.lua
#!/usr/bin/env lua

local socket = require("socket")

function run()
	local x = 0

	local start_time = socket.gettime()

	for i=1,350000001,1
		do
			flag=0
			if((i%3)==0)
				then
				flag=1
				end
			if((i%2)==0)
				then
				flag=1
				end
			if(flag==0)
				then
					x=x+i
				end
	end

	local end_time = socket.gettime()

	return string.format("%d, %0.6f", x, end_time - start_time)
end

print(run())

File: mtt.rs
fn run() -> String
{
    let mut x: u64 = 0;
    let start_time = std::time::Instant::now();

    for i in 1..350000001
    {
        if (i%3)==0
        {
            continue;
        }
        if (i%2)==0
        {
            continue;
        }
        x+=i;
    }

    let end_time = start_time.elapsed();

    let etnorm = format!("{}.{}", end_time.as_secs(), end_time.subsec_nanos() / 1000);

    return format!("{}, {}", x, etnorm);
}

fn main()
{
    println!("{}", run());
}

